kornia/feature/laf.py

```diff
@@ -413,7 +412,7 @@ def extract_patches_simple(
     """
     KORNIA_CHECK_LAF(laf)
     if normalize_lafs_before_extraction:
-        nlaf: torch.Tensor = normalize_laf(laf, img)
+        nlaf = normalize_laf(laf, img)
```

Comment:
Ideally this should be type after line 413

Reply:
this explicit type is not even needed here